src/Server/grpc_protos/clickhouse_grpc.proto

```diff
@@ -32,7 +32,7 @@ message ExternalTable {
    // Data to insert to the external table.
    // If a method with streaming input (i.e. ExecuteQueryWithStreamInput() or ExecuteQueryWithStreamIO()) is used,
    // then data for insertion to the same external table can be split between multiple QueryInfos.
-   string data = 3;
+   bytes data = 3;
```

Comment:
The types `bytes` and `string` are translated by protobuf compiler to the same C++ type `std::string`, however they're translated to different types in some other languages (Java, Go, Python). The type `bytes` here is a bit more correct because `data` and other fields contain sequences of bytes, not characters.